edx_proctoring/tests/test_reviews.py

```diff
@@ -448,3 +448,70 @@ def test_status_reviewed_by_field(self):
         ProctoredExamReviewCallback().make_review(self.attempt, test_payload)
         review = ProctoredExamSoftwareSecureReview.objects.get(attempt_code=self.attempt['attempt_code'])
         self.assertEqual(review.reviewed_by, user)
+
+    def test_is_user_course_or_global_staff(self):
```

Comment:
This may be overkill. I'm not sure that it's best to test a utility function instead of just testing the function that uses it directly. However, I do think this is clearer than combining the cases in this test with the ones in `test_reviewed_by_is_course_or_global_staff`.